packages/application-shell/src/components/theme-switcher/theme-switcher.tsx

```diff
@@ -1,13 +1,36 @@
+import { useState } from 'react';
 import { useFeatureToggle } from '@flopflip/react-broadcast';
 import { themesOverrides } from '@commercetools-frontend/application-components';
 import { ThemeProvider } from '@commercetools-uikit/design-system';
+import { STORAGE_KEYS } from '../../constants';
 import { UI_REDESIGN } from '../../feature-toggles';
+import NewDesignReleaseInfoDialog from './new-design-release-info-dialog';
```

Comment:
Nit: should we lazy load this, to avoid having it included in the main bundle since it's effectively only used once?